madminer/core/madminer.py

```diff
@@ -216,7 +216,9 @@ def add_benchmark(self, parameter_values, benchmark_name=None, verbose=True):
 
         # Default names
         if benchmark_name is None:
-            benchmark_name = "benchmark_" + str(len(self.benchmarks))
+            benchmark_name = "benchmark_" + str(len(self.benchmarks))	order : 'LO' or 'NLO', optional
+            Differentiates between LO and NLO order runs. Minor changes to writing, reading and naming cards.
```

Comment:
Huh, was this line meant to be in the middle of code (outside a docstring)?

Reply:
Ah, sorry, that must have been my rather sensitive scroll wheel inserting copies - did not notice that.
Should be alright now